x/pairing/keeper/keeper.go

```diff
@@ -61,3 +61,10 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) IsEpochStart(ctx sdk.Context) (res bool) {
```

Comment:
drop this, and have caller use epochstorage's IsEpochStart()

Reply:
done